Python/Salt n pepper noise removal/salt n pepper.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[2]:
+
+
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+
+# get image
+img = cv2.imread("gaussian Noise.png", 0)
+
+kernel = np.ones((5,5), np.float32)/25
+dst = cv2.filter2D(img, -1, kernel) #source, desirable depth, kernel
+
+# both printing together
+# result = np.hstack((img,dst))
+
+plt.imshow(cv2.cvtColor(img,cv2.COLOR_BGR2RGB))
+plt.show()
+
+plt.imshow(cv2.cvtColor(dst,cv2.COLOR_BGR2RGB))
+plt.show()
+
+
+# In[9]:
+
+
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+
+# load image
+img = cv2.imread("gaussian Noise.png", 0)
```

Comment:
no file

Reply:
Did you check my PR fully? I have already added these files.